csrankings-f.csv

```diff
@@ -163,6 +166,7 @@ Fei Miao,University of Connecticut,http://feimiao.org,fH2YF6YAAAAJ
 Fei Peng,Hunan University,http://csee.hnu.edu.cn/people/pengfei,FT0YdtIAAAAJ
 Fei Sha,University of Southern California,http://www-bcf.usc.edu/~feisha,8rttV7kAAAAJ
 Fei Song,University of Guelph,https://www.uoguelph.ca/computing/people/fei-song,NOSCHOLARPAGE
+Fei Su,BUPT,https://teacher.bupt.edu.cn/sufei/en/index/36998/list/index.htm,NOSCHOLARPAGE

```

Comment:
Not clearly in CS; please clarify.

Reply:
Fei Su is also in Artificial Intelligence department, which is belong to School of CS. She advises Phd students to publish papers at Computer Vision conference.
Z. Meng, Z. Zhao, **F. Su(( and L. Guo, "Hierarchical Spatial Pyramid Network For Cervical Precancerous Segmentation By Reconstructing Deep Segmentation Networks," 2021 IEEE/CVF Conference on Computer Vision and Pattern Recognition